http-endpoint-example/host.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import json
+import os.path
+import requests
+
+from flask import Flask, request
+# pprint used to make the output easier to read
+from pprint import pprint
```

Comment:
This is still a built-in module, so should be in the first section. It'd be tidier to just use `import pprint` instead of the `from ...` syntax.